src/app/store.ts

```diff
@@ -2,14 +2,16 @@ import { createStore, applyMiddleware, combineReducers } from 'redux';
 import { composeWithDevTools } from 'redux-devtools-extension';
 import thunk from 'redux-thunk';
 
-import currentTodoReducer from '../features/currentTodo';
-import filterReducer from '../features/filter';
-import todosReducer from '../features/todos';
+import { allTodosReducer } from '../features/todos';
+import { filterReducer } from '../features/filter';
+import { todosReducer } from '../features/activeTodo';
+import { userActiveReducer } from '../features/activeUser';
 
 const rootReducer = combineReducers({
-  currentTodo: currentTodoReducer,
+  activeTodo: todosReducer,
```

Comment:
```suggestion
  activeTodo: activeTodoReducer,
```
you handle only one (active) todo, so todo**s** in plural could be confusing